automated_packaging/update_cloudformation.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+
+$NEW_VERSION=$ARGV[0];
+$OLD_VERSION=$ARGV[1];
+
+`git checkout master`;
+
+# Update configuration file
+$release_major = substr($OLD_VERSION, 0, 1);
+`sed -i -- 's/$OLD_VERSION/$NEW_VERSION/g' citus-$release_major/citusdb.json`;
+
+# Commit changes to github
+`git commit -a -m "Bump Citus version to $NEW_VERSION"`;
+`git push origin master`;
+
+# Push the new json template to S3
+`aws s3 cp --acl public-read --region us-east-1 citusdb.json s3://citus-deployment/aws/citus$release_major/cloudformation/citus-$NEW_VERSION.json`;
```

Comment:
How did we authenticate for s3? Also similar to above comment, we need to create necessary directory structure for major versions in s3.

Reply:
Utilizing AWS CLI configuration tuned by user beforehand